### ISSUE TITLE
Search Page UX Update

### ISSUE THREAD

**Issue by TheRealArtengine** [cubeca/cube#417]
Further to the process of finding our profile page. The Search still seems awkward.

1) From clicking on Search it takes me to the search page but it says "Welcome" which is confusing.  Take out the word welcome. 

2) Also there should be no scrolling on the search page to get to the search bar. I would suggest moving it up above the highlights gallery, so it is immediately accessible.

3) I think we should make the search input a different colour because it is hard to see. The Search bar itself is not too visible. It looks very similar to text and headings.

**Comment by AshBC** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Wednesday, 02-Oct-24 20:32:41 UTC` by Gov4Git `dev`

#### Notice `twzbv2`
Started managing this issue, using the Open-Source Management Protocol (Waimea), as Gov4Git concern `417` with initial __priority score__ of `0.000000`.

This project is managed by [Gov4Git](https://gov4git.org), a decentralized governance system for collaborative git projects.
To participate in governance, __install the [Gov4Git desktop app](https://github.com/gov4git/desktop-application)__.
	

#### Notice `ipp4bd`
This issue's __priority score__ is now `0.000000`.
The __cost of priority__ is `0.000000`.
The __projected bounty__ is now `0.000000`.

#### Notice `eff5nn`
The set of eligible proposals claiming this issue is empty.

**Comment by AshBC** [comment]
For Accessibility Testing we actually already implemented :

#1 "remove the word Welcome from the search Page" 

& for #3 we updated the button to the search bar in the nave bar to a search icon and the word content. In the search bar we removed the placeholder text which was deemed not accessible and used the icon to also address visual description. Creating a lighter background was also recommended by Hello Access.

Really we are voting on #2

**Comment by AshBC** [comment]
This is zero dev hours - low to zero cost for change

**Comment by AshBC** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Friday, 29-Nov-24 06:51:14 UTC` by Gov4Git `dev`

#### Notice `rjkknm`
This issue's __priority score__ is now `1.000000`.
The __cost of priority__ is `1.000000`.
The __projected bounty__ is now `2.000000`.

#### Notice `scpziw`
The set of eligible proposals claiming this issue is empty.

**Comment by AshBC** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Friday, 29-Nov-24 16:14:32 UTC` by Gov4Git `dev`

#### Notice `hjz5g3`
This issue's __priority score__ is now `3.000000`.
The __cost of priority__ is `5.000000`.
The __projected bounty__ is now `6.000000`.

#### Notice `ubiixa`
The set of eligible proposals claiming this issue is empty.

**Comment by mikestjean** [comment]
I agree moving the search bar up to below 'Search Content | Rechercher notre contenu', for example, would make a lot more sense. It is confusing to click the 'Search' icon and then arrive at a page without a Search bar in view.

**Comment by AshBC** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Friday, 29-Nov-24 18:36:03 UTC` by Gov4Git `dev`

#### Notice `yhhnv3`
This issue's __priority score__ is now `8.000000`.
The __cost of priority__ is `14.000000`.
The __projected bounty__ is now `16.000000`.

#### Notice `x3u3yv`
The set of eligible proposals claiming this issue is empty.

**Comment by AshBC** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Tuesday, 03-Dec-24 19:59:04 UTC` by Gov4Git `dev`

#### Notice `rp46tl`
This issue's __priority score__ is now `11.000000`.
The __cost of priority__ is `19.000000`.
The __projected bounty__ is now `22.000000`.

#### Notice `e76wcz`
The set of eligible proposals claiming this issue is empty.

**Comment by AshBC** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Wednesday, 04-Dec-24 02:44:15 UTC` by Gov4Git `dev`

#### Notice `5cznl3`
This issue's __priority score__ is now `13.000000`.
The __cost of priority__ is `23.000000`.
The __projected bounty__ is now `26.000000`.

#### Notice `7xjcdg`
The set of eligible proposals claiming this issue is empty.

**Comment by AshBC** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Thursday, 12-Dec-24 11:01:00 UTC` by Gov4Git `dev`

#### Notice `guuwi3`
This issue's __priority score__ is now `12.000000`.
The __cost of priority__ is `22.000000`.
The __projected bounty__ is now `24.000000`.

#### Notice `pyxrqs`
The set of eligible proposals claiming this issue is empty.
